train.py

```diff
@@ -201,18 +222,31 @@ def train():
         args.resume = SavePath.get_interrupt(args.save_folder)
     elif args.resume == 'latest':
         args.resume = SavePath.get_latest(args.save_folder, cfg.name)
-
-    if args.resume is not None:
-        print('Resuming training, loading {}...'.format(args.resume))
-        yolact_net.load_weights(args.resume)
-
+    elif str2bool(args.resume):
+        if is_valid_stub(args.ckpt):
+            args.resume = download_checkpoint_from_stub(args.ckpt)
+        else:
+            args.resume = args.ckpt
+
+    if args.resume and args.resume.lower() not in ("no", "false", "f", "0"):
+        print('Resuming training, loading checkpoint {}...'.format(args.resume))
+        checkpoint_epoch, checkpoint_recipe, sparseml_wrapper = yolact_net.load_checkpoint(args.resume, args.recipe, resume=args.cont)
+        start_epoch = 0
+        if checkpoint_epoch and checkpoint_recipe and args.cont:
```

Comment:
let's remove `checkpoint_recipe` as a condition here since we're just dealing with epoch